openapi/kas-fleet-manager-private.yaml

```diff
@@ -258,7 +258,8 @@ components:
                       type: string
                     bf2.org/kafkaInstanceProfileQuotaConsumed:
                       type: string
-
+                    bf2.org/deployment:
```

Comment:
@shawkins as described in the epic brief:
The label `bf2.org/deployment` will be set for `reserved` kafkas only. For real workload this label won't be set